docs/features/custom_configuration.md

```diff
@@ -0,0 +1,30 @@
+# Custom configuration
+
+You can override some default properties if your environment requires that.
+
+## Configuration locations
+The configuration will be loaded from multiple locations. Properties are considered in the following order:
```

Comment:
Looking at the code, I think tc-go doesn't necessarily follow this order. I think I'd need to have some actual tests to really understand the effective order.